lib/ecto/repo.ex

```diff
@@ -405,8 +405,8 @@ defmodule Ecto.Repo do
   ## Examples
 
       MyRepo.transaction(fn ->
-        MyRepo.update(%{alice | balance: alice.balance - 10})
-        MyRepo.update(%{bob | balance: bob.balance + 10})
+        MyRepo.update(%{alice | balance: fragment("? - 10", alice.balance)})
```

Comment:
We shouldn't change those because those are not queries, just straight Elixir code. :)